common/src/main/java/com/scottlogic/deg/common/validators/ValidationResult.java

```diff
@@ -0,0 +1,48 @@
+package com.scottlogic.deg.common.validators;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ValidationResult
+{
+    public final boolean isValid;
+    public final List<String> errors;
```

Comment:
In general, exposing public fields is dangerous. It allows arbitrary access and modification the `List<T>`.

Reply:
Noted, I will revisit this in the validation pull request